src/pages/Facility/settings/layout.tsx

```
@@ -1,5 +1,7 @@
-import { useRoutes } from "raviger";
 
+// create a layout for the facility settings page
+import { Link, navigate, useRoutes } from "raviger";
```

ESLint

[error] 'Link' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)